iotdb-core/consensus/src/main/java/org/apache/iotdb/consensus/iot/service/IoTConsensusRPCServiceProcessor.java

```diff
@@ -115,8 +115,11 @@ public void syncLogEntries(
                             : ByteBufferConsensusRequest::new)
                     .collect(Collectors.toList())));
       }
+      long buildRequestTime = System.nanoTime();
       IConsensusRequest deserializedRequest =
           impl.getStateMachine().deserializeRequest(logEntriesInThisBatch);
+      impl.getIoTConsensusServerMetrics()
```

Comment:
Do we need to divide by the number of logs?

Reply:
Yes.